OmniKit/PumpManager/OmnipodPumpManager.swift

```diff
@@ -722,18 +722,18 @@ extension OmnipodPumpManager {
         }
     }
 
-    public func refreshStatus(completion: ((_ result: PumpManagerResult<StatusResponse>) -> Void)? = nil) {
+    public func refreshStatus(userRequest: Bool, completion: ((_ result: PumpManagerResult<StatusResponse>) -> Void)? = nil) {
```

Comment:
Rather than making PumpManager aware of the use case, I'd rather expose the meaning of what does to the caller, so `userRequest` -> `emitConfirmationBeeps`.  Logic doesn't need to change, but it lets the caller decide when it's appropriate to beep.

Reply:
Renamed userRequest parameter to expose the meaning as requested